src/main/java/com/ldtteam/structurize/items/ModItems.java

```diff
@@ -16,12 +19,18 @@
     private ModItems() { /* prevent construction */ }
 
     private static final DeferredRegister<Item> ITEMS = DeferredRegister.create(ForgeRegistries.ITEMS, Constants.MOD_ID);
+    private static final DeferredRegister<Item> VANILLA_ITEMS = DeferredRegister.create(ForgeRegistries.ITEMS, "minecraft");
 
     public static DeferredRegister<Item> getRegistry()
     {
         return ITEMS;
     }
 
+    public static DeferredRegister<Item> getVanillaOverrideRegistry()
```

Comment:
this class is def missing jdoc too

Reply:
class?

Reply:
I marked it here a sone of the examples, but I meant all changes in this class =D

Reply:
bleh, this class never had jdoc :P
also obsolete as we decided to have air items in our namespace

Reply:
Yep